includes/abstracts/class-llms-rest-posts-controller.php

```diff
@@ -885,6 +885,8 @@ protected function prepare_items_query_orderby_mappings( $query_args, $request )
 	 */
 	protected function prepare_item_for_database( $request ) {
 
+		$prepared_item = [];
```

Comment:
we don't use `[]` as a shorthand for `array()` in LifterLMS. Nor do we use `[ 'item', 'item2', 'etc' ]`

We do allow pushing elements into an array like `$arr = array(); $arr[] = 'item';`

@eri-trabiccolo has been trying to convince me to change (and I'd like to) but at this moment we're not using this syntax (as well as a few other PHP 5.4+ syntax allowances.

We could start doing it without issue but I don't want an anymore schizophrenic codebase than we already have. At some point in the future we'll make a change and do some blanket conversions but for now please adjust to `$prepared_item = array()`.

Thank you for your understanding